example/provider-local/seed-kind-ha/local-mz/seed.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: core.gardener.cloud/v1beta1
+kind: Seed
+metadata:
+  name: local
```

Let's maybe call it `local-ha` so that it's clear right away?
```suggestion
  name: local-ha
```

But see above, you should better use a patch file here to change the name, ingress domain, secret ref, and spec.highAvailability: https://github.com/gardener/gardener/blob/66b1975adf5d6656930d1273c150792028646ad5/example/provider-local/seed-kind2/local/patch-seed.yaml#L1-L9